test-requirements.txt

```diff
@@ -38,7 +38,9 @@ easy-vault>=0.7.0
 easy-server>=0.8.0
 pytest-easy-server>=0.8.0
 jsonschema>=2.6.0,!=4.0.0; python_version <= '3.6'
-jsonschema>=4.17.3; python_version >= '3.7'
+jsonschema>=4.17.3; python_version == '3.7'
+# jsonschema v 4.18.0 required by jupyter-events 0.9.1
+jsonschema==4.18.0; python_version >= '3.8'
```

Comment:
This should be "jsonschema>=" instead of "jsonschema==".

Reply:
Arrrrgh. age issue.  me.   Done